### PR TITLE
Sync template

### PR DESCRIPTION
**Pull request by Keyrxng**
Resolves https://github.com/ubiquity/pay.ubq.fi/issues/166

### CODE REVIEW

**Comment by Keyrxng** [comment]
everything resolved but haven't fully tested all of the functions

**Comment by 0x4007** [comment]
Thank for this! But you should have used @ubiquity/pay.ubq.fi as the merge target.

File: static/scripts/rewards/web3/wallet.ts [hunk 1]
```diff
@@ -31,10 +34,10 @@ export async function handleNetwork(desiredNetworkId: number) {
     invalidateButton.disabled = true;
   }
 
-  const currentNetworkId = parseInt(await web3provider.provider.request!({ method: "eth_chainId" }), 16);
+  const currentNetworkId = parseInt(String(web3provider.network.chainId), 16);
```
Comment (0x4007) [inline static/scripts/rewards/web3/wallet.ts]: Seems suspicious.

File: static/scripts/rewards/web3/wallet.ts [hunk 2]
```diff
@@ -11,13 +11,16 @@ export async function connectWallet(): Promise<JsonRpcSigner | null> {
...
+  } catch (error: unknown) {
+    if (error instanceof Error) {
+      if (error?.message?.includes("missing provider")) {
+        toaster.create("info", "Please use a web3 enabled browser to collect this reward.");
+        claimButton.element.disabled = true;
+      } else {
+        toaster.create("info", "Please connect your wallet to collect this reward.");
+        claimButton.element.disabled = true;
+      }
+      return null;
```
Comment (0x4007) [inline static/scripts/rewards/web3/wallet.ts]:
```suggestion

```

File: static/scripts/rewards/abis/nftRewardAbi.ts [hunk 3]
```diff
@@ -1,3 +1,4 @@
+/* eslint-disable sonarjs/no-duplicate-string */
```
Comment (0x4007) [inline static/scripts/rewards/abis/nftRewardAbi.ts:1]: What's this for?

File: static/scripts/audit-report/types/audit.d.ts [hunk 4]
```diff
@@ -119,7 +119,19 @@ interface TableIndex {
...
+    ? number
+    : NonNullable<unknown> extends boolean
+    ? boolean
+    : NonNullable<unknown> extends Date
+    ? Date
+    : NonNullable<unknown> extends object
+    ? object
+    : NonNullable<unknown> extends Array<unknown>
+    ? Array<unknown>
+    : never;
```
Comment (0x4007) [inline static/scripts/audit-report/types/audit.d.ts]: There must be a simpler type for this?

File: .eslintrc
Comment (0x4007) [inline .eslintrc:1]: Seems unusual to me that these were formatted differently.

**Review by 0x4007 — CHANGES_REQUESTED** [review]
Looks great but I saw some words that I don't expect to have passed spell check. Did you make sure to run `cspell` as part of the `yarn format` and commit lint?

File: static/scripts/rewards/abis/nftRewardAbi.ts (hunk 3 above)
Comment (0x4007) [reply static/scripts/rewards/abis/nftRewardAbi.ts:1]: I see. Redundant strings here.

**Pull request merged**